kratos/containers/pointer_vector_set.h

```diff
@@ -448,8 +386,8 @@ class PointerVectorSet final
 
     iterator erase(iterator pos)
     {
-		if (pos.base() == mData.end())
-			return mData.end();
+	if (pos.base() == mData.end())
+	    return mData.end();
         iterator new_end = iterator( mData.erase( pos.base() ) );
         mSortedPartSize = mData.size();
```

Comment:
```suggestion
	    if (pos.base() == mData.end())
	        return mData.end();
        iterator new_end = iterator( mData.erase( pos.base() ) );
        mSortedPartSize = mData.size();
```
Misleading indentation.